backend/db/migrate/20210411020529_add_restaurants_table.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# This migration will add a restaurants table
+class AddRestaurantsTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name, null: false
+      t.string :location, null: false
+      t.text :description, null: false
+    end
```

Comment:
Should there be a `creator_id` and `creator` association? It could be left for the next PR (restaurant creation), but since it's kind of a fundamental association, it seems worth adding now instead of later.